packages/lms/src/lib/components/content/Main.svelte

```diff
@@ -13,4 +13,19 @@
 		grid-area: body;
 		overflow-y: auto;
```

Comment:
We could add this to the main css to at least change the colour in firefox
`scrollbar-color: var(--primary) var(--background-primary);`

Reply:
Done :)